test/data/event/interactiveReady.html

```diff
@@ -1,24 +1,23 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<!DOCTYPE html>
 <html>
 <head>
-<meta http-equiv="content-type" content="text/html; charset=utf-8">
-<title>Test case for gh-2100</title>
-<script src="../../jquery.js"></script>
-<script src="../iframeTest.js"></script>
+	<meta charset="UTF-8">
+	<title>Test case for gh-2100</title>
+	<script src="../../jquery.js"></script>
+	<script src="../iframeTest.js"></script>
 </head>
 <body>
+	<script type="text/javascript">
+	jQuery( function () {
+		startIframeTest( jQuery("#container").length === 1 );
+	} );
+	</script>
 
-<script type="text/javascript">
-jQuery( function () {
-	startIframeTest( jQuery("#container").length === 1 );
-} );
-</script>
-
-<!-- external resources that come before elements trick
-	oldIE into thinking the dom is ready, but it's not...
-	leaving this check here for future trailblazers to attempt
-	fixing this...-->
-<script type="text/javascript" src="../longLoadScript.php?sleep=1"></script>
-<div id="container" style="height: 300px"></div>
+	<!-- external resources that come before elements trick
+		oldIE into thinking the dom is ready, but it's not...
+		leaving this check here for future trailblazers to attempt
+		fixing this...-->
+	<script type="text/javascript" src="../longLoadScript.php?sleep=1"></script>
+	<div id="container" style="height: 300px"></div>
 </body>
-</html>
+</html>
```

Comment:
Please don't remove an empty line at the end of the file. If it's missing, Git diffs are worse.